### DISCUSSION TITLE
Programming virgin rlink

### DISCUSSION THREAD

**Discussion by aberimen** [cedricp/ddt4all#1237] [Q&A]
Hello,

I bought virgin LANR16 rlink. I haven't installed it yet. What do I need to do after assembly, how can I program it? Is there anyone who can help? Thanks

**Comment by matzel82** [comment]
Display or ECU ?

normally only a Renault garage can program a RLink (if you have run a error-free system) than that’s the correct thing. It’s possible that could happen problems if you try it self.
